scripts/l10n/generate_po_files.py

```diff
@@ -87,16 +96,21 @@ def process_countries(self):
             po_directory.mkdir(parents=True, exist_ok=True)
             po_file_path = po_directory / f"{country_code}.po"
             if po_file_path.exists():
-                self.update_po_file(str(po_file_path), pot_file_path)
```

Comment:
Maybe if PO-file doesn't exist (this can only happen in case of newly localized country), create it from POT? And if it exist, it will be updated in the common cycle.
```
            if not po_file_path.exists():
                po_file = pofile(pot_file_path)
                po_file.metadata["Language"] = default_language
                po_file.save(po_file_path)
```

Reply:
Reworked a bit.